sources/packages/api/src/route-controllers/application/application.students.controller.ts

```diff
@@ -618,4 +620,43 @@ export class ApplicationStudentsController extends BaseController {
       }
     }
   }
+
+  /**
+   * Get application to request appeal.
+   ** Application eligible to be requested for
+   ** a change will be returned.
+   * @param applicationNumber
+   * @param userToken
+   * @returns application
+   */
+  @ApiOkResponse({
+    description: "Returns application which can be requested for change.",
+  })
+  @ApiNotFoundResponse({
+    description:
+      "Application either not found or not eligible to request for change.",
+  })
+  @Get(":applicationNumber/appeal")
+  async getApplicationToRequestChange(
+    @Param("applicationNumber") applicationNumber: string,
+    @UserToken() userToken: IUserToken,
+  ): Promise<ApplicationPrimaryDTO> {
```

Comment:
What does "Primary" mean in this context? Could the DTO be only ApplicationAppealDTO?

Reply:
I got it, can we then rename it to `ApplicationIdentifiersDTO`?